.changeset/yellow-cats-teach.md

```diff
@@ -0,0 +1,5 @@
+---
+"@tiptap/extension-font-family": patch
+---
+
+fix: #4311 - update the logic of removeEmptyTextStyle to manually handle the selection of all of the nodes within the selection to check for their marks independently
```

Comment:
Maybe we could explain a bit more what implications this fix has / what the problem was and that it is fixed now? Something like `fixed an issue where unsetting the fontFamily on a selection would remove all applied text style marks`?

Reply:
It's ok to have such a verbose changelog?

Reply:
I think it's fine, but in general I prefer to explain what actually changed instead of going to technical because most people will read the changelog looking for what they can expect is new on their ends.